quickstart/kubernetes/tutorial-configmap-rollout.md

```diff
@@ -1,154 +1,223 @@
 ---
-title: "Tutorial: Graceful app rollout"
+title: "Tutorial: Getting Started With Pulumi"
```

Comment:
Does this new title make sense in context of where this material is linked?  I recall some inbound links might position it differently?

Reply:
This should accurately reflect what people are coming here for.